tests/testthat/test-module_nested_tabs.R

```diff
@@ -148,29 +148,29 @@ out <- shiny::testServer(
   ),
   expr = {
     testthat::test_that("modules_reactive is a list of reactives", {
-      expect_is(modules_reactive, "list")
-      expect_is(modules_reactive$tab1, "reactive")
-      expect_is(modules_reactive$tab2, "reactive")
+      testthat::expect_is(modules_reactive, "list")
+      testthat::expect_is(modules_reactive$tab1, "reactive")
+      testthat::expect_is(modules_reactive$tab2, "reactive")
```

Comment:
Added some `testthat::` missing prefixes. Let me know if I should revert this change or resolve and merge :grin: